libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -53,6 +68,46 @@ export class PickingComponent extends LitElement {
   @asyncState()
   protected tabs = valueType(this.tabs$);
 
+  protected changeNumberOfPicked(event: Event): void {
+    const detail = (event as CustomEvent<ProductItemPickedEvent>).detail;
+
+    this.pickingList?.items.map((item) => {
+      if (
```

Comment:
this is working not properly, this changing elements of reactive array doesn't trigger update. You need to ensure whole approach